.github/workflows/os.yml

```diff
@@ -0,0 +1,92 @@
+name: "OS"
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+      - stable-*.*
+  schedule:
+    # Every day at 3:30 AM UTC
+    - cron: "30 3 * * *"
+
+env:
+  DIGRAPHS_LIB: digraphs-lib-0.6
+
+jobs:
+  test-unix:
+    name: "${{ matrix.os }}${{ matrix.ABI }} / GAP master"
+    runs-on: "${{ matrix.os }}-latest"
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu
+            ABI: 32
+            pkgs-to-clone: ""
+            pkgs-to-build: "io orb datastructures grape"
+          - os: macos
+            ABI: ""
+            pkgs-to-clone: "NautyTracesInterface"
+            pkgs-to-build: "io orb datastructures grape NautyTracesInterface"
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: "Install dependencies"
+        if: ${{ runner.os == 'macOS' }}
+        run: brew install automake
+      - name: "Install GAP and clone/compile necessary packages"
+        uses: gap-actions/setup-gap@v2
+        with:
+          GAP_PKGS_TO_CLONE: ${{ matrix.pkgs-to-clone }}
+          GAP_PKGS_TO_BUILD: ${{ matrix.pkgs-to-build }}
+          GAPBRANCH: master
+          ABI: ${{ matrix.ABI }}
```

Comment:
When we aren't explicitly testing for compatibility with a version of `GAP`, which branch should we use? Here we use `master`, but  in `config_options.yml` we use `stable-4.13`.